src/lib/ListMetrics.js

```diff
@@ -237,6 +241,10 @@ export default class ListMetrics {
                 }
             });
 
+            if (isAnyMetricNegative) {
+                return cb(errors.InternalError.customizeDescription(
+                    'Redis server is not ready'));
```

Comment:
Instead of saying `Redis server is not ready`, providing more context would be beneficial. Something like
```
Utapi is in transient state for this time period as metrics are being collected. Please try again in a few minutes.
```

Reply:
Updated.